packages/serverpod/lib/src/server/websocket_request_handlers/helpers/method_stream_manager.dart

```diff
@@ -431,6 +438,9 @@ class MethodStreamManager {
     for (var streamParam in callContext.inputStreams) {
       var parameterName = streamParam.name;
       var controller = StreamController(onCancel: () async {
+        callContext.endpoint.pod.logVerbose(
```

Comment:
Question: Same suggestion here as above: should these be logged for the user?

A tricky thing here is that the session can be closed as a result of closing the output stream. So if we want to log something for the user we need to be mindful about that.